src/ekka.appup.src

```diff
@@ -1,34 +1,56 @@
 %% -*- mode: erlang -*-
-{"0.8.1.3",
-  [{"0.8.1.2",
+{"0.8.1.4",
```

Comment:
Curiosity: are these changes manually crafted? Or is there some kind of script that generate these?

Reply:
I think it's manual